src/index.js

```diff
@@ -1,11 +1,13 @@
 import { instanceOfMainClass } from './mainClass';
-import { searchCategory, searchLinks, displayLinksForChosenCategory } from "./displayUI";
+import { renderCategories, renderLinks, searchBarCategories, searchBarLinks } from "./displayUI";
 
-instanceOfMainClass.setOnCategoriesReceived(searchCategory);
-instanceOfMainClass.setOnLinksReceived(searchLinks);
-instanceOfMainClass.setDisplayChosenLinks(displayLinksForChosenCategory);
+instanceOfMainClass.setOnCategoriesReceived(renderCategories);
+instanceOfMainClass.setOnLinksReceived(renderLinks);
 
 // Initial call of method displayCategories() in order to fill and transform this.categories;
 instanceOfMainClass.collectAllCategories();
 // Initial call of method collectAllLinks() in order to fill and transform this.links;
-instanceOfMainClass.collectAllLinks();
\ No newline at end of file
+instanceOfMainClass.collectAllLinks();
+
+searchBarCategories();
```

Comment:
On this manner implement also this:
// initialization of btnClear for Category
document.querySelector("#btnClear").addEventListener("click", (e) => {
  e.preventDefault();
  btnClear();
});

// initialization of btnClear for Links
document.querySelector("#btnClearLinks").addEventListener("click", (e) => {
  e.preventDefault();
  btnClearLinks();
});
as for search bar